content/blog/2017-06-21-prometheus-20-alpha3-new-rule-format.md

```diff
@@ -0,0 +1,71 @@
+---
+ title: Prometheus 2.0 Alpha.3 with New Rule Format
+ created_at: 2017-06-21
+ kind: article
+ author_name: Goutham Veeramachaneni
+---
+
+Today we release the third alpha version of Prometheus 2.0. Aside from a variety of bug fixes in the new storage layer, it contains a few planned breaking changes.
+ 
+First, we moved to a new flag library, which uses the more common double-dash `--` prefix for flags instead of the single dash Prometheus used so far. Deployments have to be adapted accordingly.
+Additionally, further flags were removed with this alpha. The full list since Prometheus 1 is:
+ 
+`web.telemetry-path`<br/>
+All `storage.remote.*` flags<br/>
```

Comment:
Uhm good question. I thought those were all legacy and done in the config file now but haven't verified that. Some of the timeouts etc. might have stayed the same.